### DISCUSSION TITLE
Thunderbird Add-on

### DISCUSSION THREAD

**Discussion by torgeros** [tutao/tutanota#6487] [Q&A]
Is there any development going on, either in this organization or externally, for a Thunderbird Add-on that supports Tuta?

Is this a planned thing?

What are the facts opposing the creation of such an Add-on?
I know that creating a local inbox copy is not a great idea for security reasons, but could it be possible to create something in Thunderbird without such a local copy?

**Comment by Kubunter** [comment]
Maybe this is what suits your needs: https://github.com/tdmrhn/Thunderbird-Quick-Access-Buttons
There is a Tuta-Button available.
